Source/Scene/CreditDisplay.js

```diff
@@ -66,8 +66,16 @@ function createCreditElement(element, elementWrapperTagName) {
 function displayCredits(container, credits, delimiter, elementWrapperTagName) {
   const childNodes = container.childNodes;
   let domIndex = -1;
+
+  // Sort the credits such that more frequent credits appear first
+  const creditsSorted = credits.slice();
+  creditsSorted.sort(function (credit1, credit2) {
+    return credit2[1] - credit1[1];
+  });
+
   for (let creditIndex = 0; creditIndex < credits.length; ++creditIndex) {
-    const credit = credits[creditIndex];
+    const creditInfo = creditsSorted[creditIndex];
+    const credit = creditInfo[0];
```

Comment:
I like the tuple concept but it can be easy to lose track of which array element does what. Maybe a struct would be better? Something like

```js
function CreditDisplayElement(credit) {
    this.credit = credit;
    this.count = 0;
}
```